tests/framework/util.go

```diff
@@ -148,3 +137,72 @@ func RemoveDataVolume(dvName string, namespace string) {
 	err = virtCli.CdiClient().CdiV1alpha1().DataVolumes(namespace).Delete(dvName, nil)
 	Expect(err).ToNot(HaveOccurred())
 }
+
+func CreateVMorVMIfromManifest(manifest, nameSpace string) string {
+
+	//workaround (maybe some kind of dev env issue)
+	_, _, err := ktests.RunCommandWithNS(nameSpace, "oc", "project", nameSpace)
+	Expect(err).ToNot(HaveOccurred())
+	outputVMcreate, _, err := ktests.RunCommandWithNS(nameSpace, "oc", "create", "-f", manifest)
+	Expect(err).ToNot(HaveOccurred())
+
+	var outputVMstring = string(outputVMcreate)
+	//fmt.Println("pwd=",outputVMstring)
+	vmCreatedSplit := strings.Split(outputVMstring, " ")
+	vmNameSplit := strings.Split(vmCreatedSplit[0], "/")
+	vmName := vmNameSplit[1]
+	//        ktests.WaitUntilVMIReady(vmName, ktests.LoggedInAlpineExpecter)
+
+	//	getVMOptions := metav1.GetOptions{}
+	//	vm, err := virtClient.VirtualMachine(ktests.nameSpace).Get(vmName, &getVMOptions)
+	//	Expect(err).ToNot(HaveOccurred())
+	return vmName
+	//fmt.Println("vmname=","_",vmName)
+}
+
+// Find out how many VMs with specific resource needs can be run in the cluster
+
+func min(a, b int) int {
+	if a < b {
+		return a
+	}
+	return b
+}
+
+func GetAvailableResources(virtClient kubecli.KubevirtClient, cpuNeeded int64, memNeeded int64) (int, int, int) {
+	nodeList := ktests.GetAllSchedulableNodes(virtClient)
+	availableVMs, cpu_limit_total, mem_limit_total := 0, 0, 0
+
+	for _, node := range nodeList.Items {
+		cpu := node.Status.Allocatable["cpu"]
+		mem := node.Status.Allocatable["memory"]
+		available_cpu, CpuOK := (&cpu).AsInt64()
+		available_mem, MemOK := (&mem).AsInt64()
+		if CpuOK && MemOK {
+			cpu_limit := int(available_cpu / cpuNeeded)
+			mem_limit := int(available_mem / memNeeded)
+			cpu_limit_total += cpu_limit
+			mem_limit_total += mem_limit
+			availableVMs += min(cpu_limit, mem_limit)
+		}
+	}
+
+	return availableVMs, cpu_limit_total, mem_limit_total
+}
+
+// add by https://github.com/kubevirt/kubevirt-ansible/pull/531/, delete from here
+
+func GetLatestGitHubReleaseURL(user_name string, repo_name string) string {
```

Comment:
why you need this ?

Reply:
I already use it in several my test cases. Useful function to download latest release of any project in the github. For example, latest release of our common-templates. I can modify it and add another option - to download specific release. Already shown it to Martin.